packages/lodestar-validator/src/validator.ts

```diff
@@ -73,6 +75,54 @@ export class Validator {
     if (this.blockService) await this.blockService.stop();
   }
 
+  /**
+   * Perform a voluntary exit for the given validator by its public key.
+   */
+  public async voluntaryExit(publicKey: string): Promise<void> {
+    await this.apiClient.connect();
+
+    const stateValidator = await this.apiClient.beacon.state.getStateValidator(
+      "head",
+      this.config.types.BLSPubkey.fromJson(publicKey)
+    );
+    if (!stateValidator) throw new Error("No validator found in validator store.");
+
+    const epoch = this.apiClient.clock.currentEpoch;
```

Comment:
I think this should be passed as argument to function (and cli) and can default to current epoch. I think it's idea that you can schedule exit in the future